src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -52,18 +52,9 @@
  * Snowflake via Sink service
  */
 public class SnowflakeSinkTask extends SinkTask {
-  // SfTask[ID:taskId.taskOpenCount, #totalTaskCreationCount]
```

Comment:
Got feedback and also personally was confused by this tag.... so making it just [TASK:taskid.startTimeHashCode]

Reply:
In the comment you mentioned 
[Task:taskId.startTime]

I am okay with keeping startTime instead of `startTimeHashCode`